core/solidus_core.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'ffaker', '~> 2.0'
   s.add_dependency 'friendly_id', '~> 5.0'
   s.add_dependency 'highline', '~> 1.7' # Necessary for the install generator
-  s.add_dependency 'kaminari', '~> 0.15', '>= 0.15.1'
+  s.add_dependency 'kaminari', '~> 1.0'
```

Comment:
https://github.com/kaminari/kaminari/blob/master/CHANGELOG.md#breaking-changes

The most concerning one is the removal of `num_pages` on AR scopes.

Thanks